client/src/components/FeedbackForm/FeedbackForm.js

```diff
@@ -30,7 +30,7 @@ const FeedbackForm = () => {
 
   const defaultValues = useMemo(
     () => ({
-      subject: 'Feedback for the Frontera Portal',
+      subject: 'Feedback for the portal',
```

Comment:
__Consistency__

```suggestion
      subject: 'Feedback for the Portal',
```

Same change in […/ManageAccount/ManageAccountForms/ChangePasswordForm.js](https://github.com/TACC/Frontera-Portal/pull/329/files#diff-d540a53f6a3e2241a02e7ce59da6fb441f7e9a8ba94e325259754dbbdda39ac0) but without the change to lowercase.